addons/web/static/src/views/fields/statusbar/statusbar_field.js

```diff
@@ -112,7 +111,7 @@ export class StatusBarField extends Component {
 
         // Command palette
         if (this.props.withCommand) {
-            const moveToCommandName = _t("Move to %s...", escape(this.field.string));
```

Comment:
Note: I don't see the point of escaping a standard text, won't this double escape when used?